src/main/resources/db/migration/V030__add_discount_eyca_flag.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE discount ADD COLUMN visible_on_eyca BOOLEAN NULL;
```

Comment:
NOT NULL i guess

Reply:
fixed in https://github.com/pagopa/cgn-onboarding-portal-backend/pull/84/commits/eea04e476a3b7f0d35f7dc15faabe462f9483ec9